services/validate.js

```diff
@@ -19,12 +17,19 @@ fs.readdirSync(baseDir)
   })
   .forEach(function (fileName) {
     try {
-      validator.addSchema(fileName, require(path.join(baseDir, fileName)))
+      validator.addSchema(JSON.parse(path.join(baseDir, fileName)), fileName)
```

Comment:
Fixed.

The ledger's unit and integration tests run smoothly when npm linking this PR. When using the branch [ilp-connector/tree/da-fix-tests-for-ajv](https://github.com/interledgerjs/ilp-connector/tree/da-fix-tests-for-ajv), the connector's tests are also working against this PR.